src/components/main-banner/index.tsx

```diff
@@ -0,0 +1,24 @@
+import { makeStyles, Title2 } from '@fluentui/react-components';
+
+export const useClasses = makeStyles({
+  main: {
+    minHeight: `calc(100vh - 100px)`,
```

Comment:
```suggestion
    minHeight: `calc(100vh - 100px)`,
```
minHeight: 'calc(100vh - 100px)',